### ISSUE TITLE
Missing packages in registry-image-helper.sh

### ISSUE THREAD

**Issue by hussam-qasem** [oracle/vagrant-projects#488]
`/usr/bin/registry-image-helper.sh` part of [`olcne-utils`](https://yum.oracle.com/repo/OracleLinux/OL8/olcne16/x86_64/index.html) package is missing configurations of extra packages for Kubernetes 1.25.7 (OCNE1.6)

For example:
```
sudo dnf install -y oracle-olcne-release-el8
sudo dnf config-manager --enable ol8_olcne16 ol8_addons ol8_baseos_latest ol8_appstream ol8_kvm_appstream ol8_UEKR7
sudo dnf config-manager --disable ol8_olcne15 ol8_olcne14 ol8_olcne13 ol8_olcne12
sudo dnf install -y olcne-utils
```

```
$ rpm -qf /usr/bin/registry-image-helper.sh
olcne-utils-1.6.0-4.el8.x86_64
```
and:

Suppose you have a local registry on 10.0.2.2:5000, and perform:
```
/usr/bin/registry-image-helper.sh --to 10.0.2.2:5000/olcne --version 1.25.7
```

This will not copy extra packages required for Kubernetes 1.25.7 (OCNE1.6), for example: Calico, Multus, etc...

The `1.25.7_extra_images` is missing. For example, it should be:
```
function 1.25.7_extra_images {
  echo "kubernetes-dashboard: v2.5.1"
  echo "flannel: v0.14.1"
  echo "nginx: 1.17.7"
  echo "externalip-webhook: v1.0.0-1"
  echo "apiserver: v3.25.0"
  echo "cni: v3.25.0"
  echo "csi: v3.25.0"
  echo "kube-controllers: v3.25.0"
  echo "node-driver-registrar: v3.25.0"
  echo "node: v3.25.0"
  echo "pod2daemon-flexvol: v3.25.0"
  echo "tigera-operator: v1.29.0"
  echo "typha: v3.25.0"
  echo "speaker: v0.12.1"
  echo "multus-cni: v3.9.3"
}
```

I opened an Oracle SR `3-32917012371` over a month ago, but the response, unfortunately, was irrelevant and does not address the issue.

**Comment by scoter-oracle** [comment]
We're aware of this issue and we're working on the same for a fix.
Will keep you posted as soon as I have updates.

**Comment by scoter-oracle** [comment]
This issue should now be fixed, please confirm.

**Comment by scoter-oracle** [comment]
Fixed in 1.6.2
